src/components/visualizations/chart/highcharts/comboConfiguration.ts

```diff
@@ -1,29 +1,46 @@
 // (C) 2007-2019 GoodData Corporation
 import get = require("lodash/get");
-import isEqual = require("lodash/isEqual");
 import { MAX_POINT_WIDTH } from "./commonConfiguration";
 import { LINE_WIDTH } from "./lineConfiguration";
 import { IChartConfig } from "../../../../interfaces/Config";
 import { VisualizationTypes } from "../../../../constants/visualizationTypes";
+import { isLineChart } from "../../utils/common";
 
 const { COLUMN, LINE } = VisualizationTypes;
 
 export function getDefaultChartType(config?: IChartConfig) {
-    const primaryType = get(config, "primaryChartType", COLUMN);
-    const secondaryType = get(config, "secondaryChartType", LINE);
+    const primaryChartType = get(config, "primaryChartType", COLUMN);
```

Comment:
what about adding a function (not tested)
```
function getDefaultComboTypes(config?: IChartConfig) {
    return {
        primaryType: get(config, "primaryChartType", COLUMN),
        secondaryType: get(config, "secondaryChartType", LINE)
    };
}

and then use on both places (here and here: https://github.com/gooddata/gooddata-react-components/pull/1025/files#diff-17f6209e1b24cc4a39babe31a1a4b51fR27) just 
`const { primaryChartType, secondaryChartType } = getDefaultComboTypes(config);`

It would be better to have "defaults" on only one place.